packages/devtools_app/lib/src/common_widgets.dart

```diff
@@ -103,7 +103,7 @@ class IconLabelButton extends StatelessWidget {
 
   final String label;
 
-  final double includeTextWidth;
+  final double unscaledIncludeTextWidth;
```

Comment:
do we need to scale this instead of leaving it unscaled? I imagine a large font size would cause overflow errors if we don't also scale this value inversely to the font size.

Reply:
got a better idea for a the variable name? What I am trying to say is this is the width you would require with the default text scale, but you will need to scale this width if using a larger font.

Reply:
some questions: What is the consumer of this widget supposed to do? If the font size is arbitrarily set by the user, then the value passed into `unscaledIncludeTextWidth` would need to be a function of the font size, right? Is this function going to be consistent for all uses of this widget, or defined on a per use basis? How can the consumer of this widget guarantee they are passing in a size that will work for all font sizes?